scripts/run_notebooks_timeouts.py

```diff
@@ -0,0 +1,103 @@
+import time
+from pathlib import Path
+
+import nbformat
+from nbconvert.preprocessors import CellExecutionError, ExecutePreprocessor
+
+TIMEOUT = 20  # seconds
+
+allow_to_fail = [
```

Comment:
Nitpick: it might be a little simpler to make these paths relative to the `notebooks` directory.